spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/reactive/DefaultPartHttpMessageReaderCustomizerConfiguration.java

```diff
@@ -0,0 +1,65 @@
+package org.springframework.boot.autoconfigure.web.reactive;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Paths;
+import java.util.function.Consumer;
+
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.context.properties.PropertyMapper;
+import org.springframework.boot.web.codec.CodecCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.annotation.Order;
+import org.springframework.http.codec.multipart.DefaultPartHttpMessageReader;
+import org.springframework.util.unit.DataSize;
+
+@Configuration(proxyBeanMethods = false)
+@EnableConfigurationProperties(MultipartProperties.class)
+class DefaultPartHttpMessageReaderCustomizerConfiguration {
+
+	@Bean
+	@Order(10)
```

Comment:
The other built-in customizers run at `@Order(0)` - I just ran this one before those and I think it really does not matter much as the setting of most fields on the Codecs cause them all to be re-inited.